packages/code-block/src/code-block.tsx

```diff
@@ -78,16 +78,24 @@ export const CodeBlock = React.forwardRef<
       `CodeBlock: There is no language defined on Prism called ${props.language}`,
     );
 
-  const lines = props.code.match(/[^\r\n]+/g) ?? [];
+  const lines = props.code.split(/\r\n|\r|\n/gm);
   const tokensPerLine = lines.map((line) =>
     Prism.tokenize(line, languageGrammar),
   );
 
   return (
-    <pre ref={ref} style={{ ...props.theme.base, ...props.style }}>
+    <pre
+      ref={ref}
+      style={{
+        ...props.theme.base,
+        maxWidth: "600px",
```

Comment:
This `maxWidth` we should debate before merging as it can be annoying to users. Without a `maxWidth` like this, I found that the preview for the **iPhone SE** was showing a code-block very stretched out. 

Here's the link to Litmus https://litmus.com/pub/nDe9bXhnjQyL9TwH

Reply:
And if the maxWidth is in the parent container of the email this still a problem?

Reply:
nope, shouldn't be